syft/formats/common/spdxhelpers/to_syft_model.go

```diff
@@ -292,6 +293,25 @@ func toSyftPackage(p *spdx.Package) *pkg.Package {
 	return &sP
 }
 
+func parseSPDXLicenses(p *spdx.Package) []pkg.License {
+	licenses := make([]pkg.License, 0)
+
+	// concluded
+	if p.PackageLicenseConcluded != NOASSERTION && p.PackageLicenseConcluded != NONE {
+		l := pkg.NewLicense(p.PackageLicenseConcluded)
+		l.Type = license.Concluded
+		licenses = append(licenses, l)
+	}
+
+	// declared
+	if p.PackageLicenseDeclared != NOASSERTION && p.PackageLicenseDeclared != NONE {
+		licenses = append(licenses, pkg.NewLicense(p.PackageLicenseDeclared))
+	}
+
+	// TODO: do we need other licenses? These are at the document level
```

Comment:
I don't understand this todo, is there a suggestion to refactor this to include all three license groups here?

Reply:
added cleaning function that removes the "License-Ref" from `other` licenses when decoding SPDX documents